.travis.yml

```diff
@@ -3,7 +3,7 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - "8"
+  - "10"
```

Comment:
Let's do this in a separate PR specifically focused on dropping node < 10.

Reply:
Definitely, together with general dependencies bump.

Reply:
PR for upgrade is here - https://github.com/adopted-ember-addons/ember-page-title/pull/170